components/byu-faculty-profile/byu-faculty-profile-common.scss

```diff
@@ -95,10 +98,11 @@
   align-content: center;
   color: #FFFFFF;
   font-family: "Gotham A", "Gotham B", Helvetica, sans-serif;;
```

Comment:
The college requested that I change the font in certain places. I've removed the extra semicolon and cleaned up the CSS, so it's a little nicer. The second font-family reference should remain A/B though. 

I've added the URL for the specs I have to the readme file (I only have a URL, not any kind of files I could upload to an actual specs folder). At this point, a lot of the changes I'm making are coming from requests at the college level though.